api/src/main/resources/org/openmrs/api/db/hibernate/Form.hbm.xml

```diff
@@ -1,83 +0,0 @@
-<?xml version="1.0"?>
-<!--
-
-    This Source Code Form is subject to the terms of the Mozilla Public License,
-    v. 2.0. If a copy of the MPL was not distributed with this file, You can
-    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
-    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
-
-    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
-    graphic logo is a trademark of OpenMRS Inc.
-
--->
-<!DOCTYPE hibernate-mapping PUBLIC
-    "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
-    "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
-<hibernate-mapping package="org.openmrs">
-
-	<class name="Form" table="form">
-
-		<id name="formId" type="int" column="form_id" unsaved-value="0">
-			<generator class="native">
-				<param name="sequence">form_form_id_seq</param>
-			</generator>
-		</id>
-
-		<property name="uuid" type="java.lang.String"
-			column="uuid" length="38" unique="true" />
-
-		<property name="name" type="java.lang.String" 
-			column="name" not-null="true" length="255" />
-
-		<property name="version" type="java.lang.String" 
-			column="version" not-null="true" length="50" />
-		
-		<property name="build" type="java.lang.Integer" column="build" 
-			not-null="false" />
-		
-		<property name="published" type="boolean" column="published"
-			length="1" not-null="false" />
-
-		<property name="description" type="java.lang.String" 
-			column="description" length="1024" />
```

Comment:
shouldn't the description be overriden as well since the length is not similar to what is defined at https://github.com/openmrs/openmrs-core/blob/39ac44536294b294a77f020c44e2725adc68cfa1/api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java#L43?